### PR TITLE
Feat: videoListing and sidebar

### PR DESCRIPTION
**Pull request by yashpatel1797**
Feat Added
1. VideoCard contains title, duration, view etc.
2. sidebar

The navbar and home page is not completed

image

### CODE REVIEW

File: src/utilities/https-helper.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
+import axios from 'axios'
+/**
+ * 
+ * @returns array which contains video data
+ */
```
Comment (rohanmathur91) [inline src/utilities/https-helper.js]: suggestion: add one-line description of what this function does

**Review by rohanmathur91 — APPROVED** [review]
- Good use of CSS modules
 - Liked that you have created an array for sidebar data and just iterating over it
 - Good use of reusable components

File: src/components/Hero/Hero.css [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+  background-position: top;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+.intro-text-box {
+  text-align: center;
+}
+.heading-primary {
+  color: #fff;
```
Comment (sharmayatendra) [inline src/components/Hero/Hero.css]: avoid using hard coded color

**Review by sharmayatendra — APPROVED** [review]
1- clean code
2- good file structure 
3- good use of CSS modules & imported from a single index file wherever required.

Added a suggestion can look into it.

File: src/components/Hero/Hero.css (hunk 2 above)
Comment (yashpatel1797) [reply src/components/Hero/Hero.css]: done

File: src/utilities/https-helper.js (hunk 1 above)
Comment (yashpatel1797) [reply src/utilities/https-helper.js]: added

**Pull request merged**